NoSuchFileException

```
[09:24:24 ERROR]: [ScriptBlock-Improved] Failed to initialize the plugin.
xyz.shirokuro.scriptblockimproved.ScriptBlockImproved$InitException: Unable to init PermissionDetector
        at xyz.shirokuro.scriptblockimproved.ScriptBlockImproved.<init>(ScriptBlockImproved.java:60) ~[?:?]
        at xyz.shirokuro.scriptblockimproved.ScriptBlockImproved.init(ScriptBlockImproved.java:97) ~[?:?]
        at xyz.shirokuro.scriptblockimproved.Bootstrap.onEnable(Bootstrap.java:12) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.12.2.jar:git-Paper-1618]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) ~[patched_1.12.2.jar:git-Paper-1618]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) ~[patched_1.12.2.jar:git-Paper-1618]
        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:395) ~[patched_1.12.2.jar:git-Paper-1618]
        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:344) ~[patched_1.12.2.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:223) ~[patched_1.12.2.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:616) ~[patched_1.12.2.jar:git-Paper-1618]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_231]
Caused by: java.nio.file.NoSuchFileException: plugins\ScriptBlock-Improved\permission-mappings.yml
        at sun.nio.fs.WindowsException.translateToIOException(Unknown Source) ~[?:1.8.0_231]
        at sun.nio.fs.WindowsException.rethrowAsIOException(Unknown Source) ~[?:1.8.0_231]
        at sun.nio.fs.WindowsException.rethrowAsIOException(Unknown Source) ~[?:1.8.0_231]
        at sun.nio.fs.WindowsFileSystemProvider.newByteChannel(Unknown Source) ~[?:1.8.0_231]
        at java.nio.file.spi.FileSystemProvider.newOutputStream(Unknown Source) ~[?:1.8.0_231]
        at java.nio.file.Files.newOutputStream(Unknown Source) ~[?:1.8.0_231]
        at java.nio.file.Files.copy(Unknown Source) ~[?:1.8.0_231]
        at xyz.shirokuro.scriptblockimproved.permission.PermissionDetector.<init>(PermissionDetector.java:54) ~[?:?]
        at xyz.shirokuro.scriptblockimproved.ScriptBlockImproved.<init>(ScriptBlockImproved.java:58) ~[?:?]
        ... 10 more
```

--- a/src/main/java/xyz/shirokuro/scriptblockimproved/permission/PermissionDetector.java
+++ b/src/main/java/xyz/shirokuro/scriptblockimproved/permission/PermissionDetector.java
@@ -51,6 +51,7 @@ public final class PermissionDetector {
             if (resource == null) {
                 throw new ResourceNotFoundException(resourceName);
             } else {
+                Files.createDirectory(filePath.getParent());
                 Files.copy(resource, filePath);
             }
         } else if (!Files.isRegularFile(filePath)) {

